src/utils/commitishFromPath.ts

```diff
@@ -1,3 +1,6 @@
 export default function commitishFromPath (path: string): string {
-  return path.split('/')[2].substr(0,8)
+	if (path) {
```

Comment:
Unsure how bad this really is, but had a lot of catastrophic failures on this because we returned bad data. 
While this is not a good fix, it does bring to light that this is one of many places where we have the frontend explode on the slightest deviation from expectations.

Reply:
interesting. this'll work for now. We should clean this up in the backend